README.md

```diff
@@ -149,13 +154,13 @@ API since digest operations do not require access to keys.
 ### SVP Operations
 
 The API provides the following SVP capabilities:
+
 * Protected buffer operations, including buffer allocation, deallocation, writes, and copies.
 * AES cipher operations on data in protected buffers using CBC and CTR mode.
 
-The initialization and configuration of the SVP is out of scope for the API. This is the
-responsibility of an SVP TA, for example. The device must provide a mechanism to the SecAPI TA to
-determine whether an SVP is enforced. The device must provide a mechanism to the SecAPI TA to
-determine whether an SVP buffer is wholly contained within the restricted SVP memory region.
+The device must provide a mechanism to the SecAPI TA to determine whether an SVP is enforced. The
```

Comment:
Use of device in this paragraph doesn't seem right. Isn't this a requirement on the SVP implementation or SOC vendor?

Reply:
Done